### PR TITLE
remove empty p

### PR DESCRIPTION
**Pull request by Fbasham**
https://dev.azure.com/VP-BD/DECD/_boards/board/t/Service%20Canada%20Labs/Stories%20and%20Activities?System.WorkItemType=Activity

There was a paragraph tag with empty content adding space.  Removed that tag.

### CODE REVIEW

**Comment by andr0272** [comment]
Check out a preview of your pull request here: https://app-servicecanadalabs-dev-pr-preview-1.azurewebsites.net

File: pages/projects/dashboard/index.js [hunk 1]
```diff
@@ -364,7 +364,7 @@ export default function MscaDashboard(props) {
                   : pageData.scFragments[3].scContentFr.json[0].content[0]
                       .value}
               </h2>
-              <p className="col-span-12 xl:col-span-8">
```
Comment (will0684) [inline pages/projects/dashboard/index.js:367]: So from what I understand, we want to try and keep our line length around 65~ characters and no more than 80~. This is why on desktop generally we have our text span 8 out of the 12 columns.

Here's the research from the DS team:

https://www.figma.com/file/XkfHmBPKDcXKkrDVIZFlyF/DS-research-(Copy)?type=whiteboard&node-id=134-10&t=XB93hQUBy11s12gu-4

**Review by will0684 — CHANGES_REQUESTED** [review]

**Comment by andr0272** [comment]
Check out a preview of your pull request here: https://app-servicecanadalabs-dev-pr-preview-1.azurewebsites.net

**Review by will0684 — APPROVED** [review]
Thank you!

**Pull request merged**